src/Symfony/Component/Messenger/Tests/Transport/AmqpExt/ConnectionTest.php

```diff
@@ -189,6 +189,36 @@ public function testItCanDisableTheSetup()
         $connection = Connection::fromDsn('amqp://localhost/%2f/messages?queue[routing_key]=my_key&auto-setup=false', [], true, $factory);
         $connection->publish('body');
     }
+
+    public function testItCanPublishWithTheDefaultQueueRoutingKey()
+    {
+        $factory = new TestAmqpFactory(
+            $amqpConnection = $this->getMockBuilder(\AMQPConnection::class)->disableOriginalConstructor()->getMock(),
```

Comment:
Why not using „createMock“ here?

Reply:
It would do the job too. I did it the same way than on the previous tests to be more predictable.

Reply:
Ok to be consistent makes sense to me.
I don’t know if Symfony has a preferred one in such case 🤔

Reply:
BTW this could be a nice job for PHP-CS-Fixer

Reply:
We use both actually. `createMock` is simpler (and used in the test above) so let's use it.